webrtc.html

```diff
@@ -1143,38 +1142,6 @@ <h4>Update the ICE gathering state</h4>
         </ol>
         </section>
 
-        <section>
-        <h4>Update the ICE connection state</h4>
```

Comment:
There is one reference to this in close(), but it doesn't give respec errors because it only used quotes:

> Set connection's ICE connection state to "closed". This does not invoke the "update the ICE connection state" procedure, and does not fire any event.